### PR TITLE
Rename using underscore

### PR DESCRIPTION
**Pull request by mjgiarlo**

### CODE REVIEW

File: lib/view_composer.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+require "view_composer/version"
+require "view_composer/base_composer"
+require 'json'
```
Comment (mjgiarlo) [inline lib/view_composer.rb:3]: Don't think this needs to be here, but is a separate concern from this PR.

**Comment by koryteg** [comment]
nice! thanks. I did a push to fix some of these on master. but if you want pull and resolve those issues I will merge this.

File: lib/view_composer.rb (hunk 1 above)
Comment (koryteg) [inline lib/view_composer.rb:3]: yeah I think we only need `require "json"` now...

**Comment by koryteg** [comment]
looks great thanks !

**Pull request merged**